simplivity/resources/policies.py

```diff
@@ -86,7 +86,7 @@ def create(self, name, flags=None, timeout=-1):
         """
         data = {"name": name}
 
-        affected_object = self._client.do_post(URL, data, timeout, flags)[0]
+        affected_object = self._client.do_post(URL, data, timeout, None, flags)[0]
```

Comment:
Can we update do_post method for having default values for custom_headers, timeout, and flags, so that we have to pass only the required parameters?

Reply:
I think we should handle this in other methods also like do_delete, do_put, etc.

Reply:
Updated the code as per the suggestion